judge/models/tests/test_contest.py

```diff
@@ -42,7 +42,7 @@ def setUpTestData(self):
             start_time=_now - timezone.timedelta(days=1),
             end_time=_now + timezone.timedelta(days=100),
             is_visible=True,
-            hide_scoreboard=True,
+            scoreboard_visibility="Contest",
```

Comment:
Ideally also add tests for the other visibility options

Reply:
I'm unsure how to simulate a user outside their contest window. There's an example for a user in a contest though.